src/components/FlawAffects/__tests__/FlawAffects.spec.ts

```diff
@@ -277,17 +288,11 @@ describe('flawAffects', () => {
     expect(resolutionSelect.text()).toBe('');
   });
 
-  osimFullFlawTest('Validates affect PURLs when present', async () => {
+  const flawInvalidPurl = structuredClone(sampleFlawFull) as ZodFlawType;
+  flawInvalidPurl.affects[0].purl = 'invalid-purl';
+  osimTestWithFlaw(flawInvalidPurl)('Validates affect PURLs when present', async () => {
```

Comment:
Is this actually necessary?
`osimTestWithFlaw` is already doing the `structuredClone` so no need to duplicate it (maybe we could remove it there instead)